MicroPython/src/communication/API_communication.py

```diff
@@ -20,30 +23,34 @@ def authorization_request() -> str:
     try:
         response = urequests.post(url, data=body, headers=headers)
     except IndexError as e:
-        logging.ingo("No internet connection")
-        return None
+        logging.info("No internet connection: {}".format(e))
+        return ""
```

Comment:
After this functon call you evaluate is by ret_val is not None to detect if valid jwt token is returned. But empty string is not None.